app/bundles/EmailBundle/DataFixtures/ORM/fakeemaildata.csv

```diff
@@ -1,2 +1,3 @@
-"subject","template","language","content","plainText","publishUp","publishDown","readCount","sentCount","revision","variantSettings","variantStartDate","variantSentCount","variantReadCount"
-"Join us at the 2014  Kaleidoscope Conference!","Mauve","en","a:2:{s:4:\"body\";s:110:\"<div>Dear {leadfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
+"name","emailType","subject","template","language","content","plainText","publishUp","publishDown","readCount","sentCount","revision","variantSettings","variantStartDate","variantSentCount","variantReadCount"
+"Kaleidoscope Conf List","list","Join us at the 2014  Kaleidoscope Conference!","Mauve","en","a:2:{s:4:\"body\";s:110:\"<div>Dear {leadfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
+"Kaleidoscope Conf Campaign","template","Join us at the 2014  Kaleidoscope Conference!","Mauve","en","a:2:{s:4:\"body\";s:110:\"<div>Dear {leadfield=firstname},</div><div>&nbsp;</div><div>Join us at the 2014 Kaleidoscope Conference!</div>\";s:6:\"footer\";s:71:\"<div>{webview_text}</div><div>&nbsp;</div><div>{unsubscribe_text}</div>\";}","Join us at the 2014  Kaleidoscope Conference!",NULL,NULL,0,0,1,"a:0:{}",NULL,0,0
```

Comment:
Please update instances of `{leadfield=` to `{contactfield=` instead, and we can get this merged. Thanks